change/@azure-msal-browser-e1f28fc4-e0a8-45ed-aca0-f6e385fe0335.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

```suggestion
  "type": "minor",
```

I think this is minor worthy